lib/controller/admin/appearance.php

```diff
@@ -254,33 +254,33 @@ protected function updateAutoTeaserConfiguration() {
 
 		if (!is_numeric($percentage) or $percentage % 1 > 0 or 1 > $percentage or $percentage > 100) {
 			$success = FALSE;
-			$errorMessages[$percentageFieldName] = __('Percentage must be a positive integer between 1 and 100', 'laterpay');
+			$errorMessages[$percentageFieldName] = tx_laterpay_helper_string::tr('Percentage must be not negative integer and have value between 1 and 100', 'laterpay');
 			$errors[] = $percentageFieldName;
 		}
 
 		if (!is_numeric($minWords) or $minWords < 0 ) {
 			$success = FALSE;
-			$errorMessages[$minWordsFieldName] = __('Min count of words must be a positive integer greater than 0', 'laterpay');
+			$errorMessages[$minWordsFieldName] = tx_laterpay_helper_string::tr('Min count of words must be not negative integer more than 0', 'laterpay');
 			$errors[] = $minWordsFieldName;
 		}
 
 		if (!is_numeric($maxWords) or $maxWords < 0 ) {
 			$success = FALSE;
-			$errorMessages[$maxWordsFieldName] = __('Max count of words must be a positive integer greater than 0', 'laterpay');
+			$errorMessages[$maxWordsFieldName] = tx_laterpay_helper_string::tr('Max count of words must be not negative integer more than 0', 'laterpay');
 			$errors[] = $maxWordsFieldName;
 		}
 
 		if ($maxWords <= $minWords) {
 			$success = FALSE;
-			$errorMessages[$maxWordsFieldName] = __('Max count of words must be greater than min count of words', 'laterpay');
+			$errorMessages[$maxWordsFieldName] = tx_laterpay_helper_string::tr('Max count of words must more than min count of words', 'laterpay');
 			$errors[] = $maxWordsFieldName;
 		}
 
 		if ($success) {
-			$message = __('New values saved successfully', 'laterpay');
-			update_option(tx_laterpay_config::REG_LATERPAY_PREVIEW_EXCERPT_PERCENTAGE_OF_CONTENT, $percentage);
-			update_option(tx_laterpay_config::REG_LATERPAY_PREVIEW_EXCERPT_WORD_COUNT_MAX, $maxWords);
-			update_option(tx_laterpay_config::REG_LATERPAY_PREVIEW_EXCERPT_WORD_COUNT_MIN, $minWords);
+			$message = tx_laterpay_helper_string::tr('new values saved successfully');
```

Comment:
Don't we need a namespace for the translations? All other translations have a namespace parameter ", 'laterpay'"